### PR TITLE
Fix npm-build-pod dockerfile

### PR DESCRIPTION
**Pull request by InfoSec812**
- Repair missing escape sequences
- Update for newer base images

### CODE REVIEW

File: docker/npm-build-pod/Dockerfile [hunk 1]
```diff
@@ -8,8 +8,8 @@ RUN yum remove -y rh-nodejs4; \
       --enablerepo=rhel-server-rhscl-7-rpms \
       --enablerepo=rhel-7-server-optional-rpms \
       --enablerepo=rhel-7-server-optional-rpms \
-      --disablerepo=epel
       $INSTALL_PKGS && \
     rpm -V $INSTALL_PKGS && \
-    yum clean all -y
+    yum clean all -y \
```
Comment (mdanter) [inline docker/npm-build-pod/Dockerfile]: is there a && missing here?

File: docker/npm-build-pod/Dockerfile [hunk 2]
```diff
@@ -8,8 +8,8 @@ RUN yum remove -y rh-nodejs4; \
...
+    rm -rf /var/cache/yum
```
Comment (mdanter) [inline docker/npm-build-pod/Dockerfile]: What is the purpose for force removing yum cache dir after issuing yum clean all -y?

File: docker/npm-build-pod/Dockerfile (hunk 1 above)
Comment (InfoSec812) [reply docker/npm-build-pod/Dockerfile]: Fixed. Thanks for the catch!

File: docker/npm-build-pod/Dockerfile (hunk 2 above)
Comment (InfoSec812) [reply docker/npm-build-pod/Dockerfile]: When you run `yum clean all -y` it actually prints a message suggesting that you ALSO do the `rm -rf /var/cache/yum`

**Comment by mdanter** [comment]
lgtm

**Pull request merged**